acra/configuring-maintaining/general-configuration/acra_server.md

```diff
@@ -0,0 +1,308 @@
+---
+title: AcraServer
+bookCollapseSection: true
+weight: 3
+---
+
+# AcraServer
+
+## Command line flags
+
+(excluding TLS-related flags that are listed [here](/acra/configuring-maintaining/tls))
+
+* `--acracensor_config_file=<filename>`
+
+  Path to AcraCensor configuration file.
+
+* `--acraconnector_tls_transport_enable={true|false}`
+
+  Enable TLS to encrypt transport between AcraConnector and AcraServer.
+  Default is `false` which means "use SecureSession instead".
+
+* `--acraconnector_transport_encryption_disable=`
+
+  Use raw transport (tcp/unix socket) between AcraTranslator and client app.
+  It turns off reading trace from client app's side which usually sent by AcraConnector.
+  Default is `false`.
+
+* `--acrastruct_injectedcell_enable={true|false}`
+
+  Acrastruct may be injected into any place of data cell.
+  Default is `false`.
+
+* `--acrastruct_wholecell_enable={true|false}`
+
+  Acrastruct will stored in whole data cell.
+  Default is `true`.
+
+* `--audit_log_enable={true|false}`
+
+  Enable audit log functionality.
+  Default is `false`.
+
+* `--auth_keys=<filename>`
+
+  Path to basic auth credentials.
+  To add user, use: `./acra-authmanager --set --user <user> --pwd <pwd>`.
+  Default is `configs/auth.keys`.
+
+* `--client_id=<id>`
+
+  Expected client ID of AcraConnector in mode without encryption.
```

Comment:
it's client_id used by AcraServer for transparent encryption in environment without acra-connector. acra-server uses this value as like it took from acra-connector.